source/win32/win_fs.c

```diff
@@ -336,7 +367,7 @@ bool Sys_FS_RemoveDirectory( const char *path )
 	WCHAR wpath[MAX_PATH];
 
 	_Sys_Utf8FileNameToWide(path, wpath, _countof(wpath));
-	return ( !_wrmdir( path ) );
+	return ( !_wrmdir( wpath ) );
```

Comment:
This line is not technically part of the fix, but it's a bugfix for the utf8 path thingy.

Reply:
Do you need to make a separate PR for this utf8 path thingy since we're no longer scanning for workshop content the same way?

Reply:
Done in #53